### PR TITLE
[20.01] Bump cryptography dependency to 3.2.1

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

**Review by nsoranzo — APPROVED** [review]

**Comment by mvdbeek** [comment]
Fun, a deprecation warning is leaking through to the job stderr.

File: lib/galaxy/metadata/set_metadata.py [hunk 1]
```diff
@@ -15,6 +15,12 @@
 import os
 import sys
 import traceback
+import warnings
+
+# Warning on stderr will cause some jobs to be failed.
+# This silences deprecation warnings in the cryptography module.
+if sys.version_info.major <= 3 and sys.version_info.minor < 6:
```
Comment (nsoranzo) [inline lib/galaxy/metadata/set_metadata.py]: Should we make this unconditional, i.e. remove this line?

Comment (mvdbeek) [reply lib/galaxy/metadata/set_metadata.py]: We could, but this isn't the right fix, unfortunately.

**Comment by mvdbeek** [comment]
API, Selenium and framework tests passed, but Jenkins didn't update the PR status.

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**